docs/resources/compute_instance.md

```diff
@@ -369,6 +369,15 @@ The following arguments are supported:
 
   -> **NOTE:** The `power_action` is a one-time action.
 
+* `auto_terminate_time` - (Optional, String) Specifies the auto terminate time.
+  The value is in the format of "yyyy-MM-ddTHH:mm:ssZ" in UTC+0 and complies with ISO8601.
+  If the value of second (ss) is not "00", the system automatically sets to the current value of minute (mm).
```

Comment:
If the second is not "00", is there a terraform plan to update?

Reply:
no ,the value is not change. it mean system will termiate at minute(mm). The second is not use.